test/e2e/performanceprofile/testdata/render-expected-output/manual_machineconfig.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
     machineconfiguration.openshift.io/role: worker-cnf
   name: 50-performance-manual
   ownerReferences:
-  - apiVersion: "performance.openshift.io/v2"
+  - apiVersion: performance.openshift.io/v2
     kind: PerformanceProfile
     name: manual
-    uid: ""
+    uid: 849928a9-63c5-4988-805a-1232f6067b89
```

Comment:
Will the UID be stable for CI? In other words, is the end result the same every run?

Reply:
hmm, it  seems like the way render command works has changed and now it generates a uid as well. I dunno if that's intended but it should not be part of the manifests for the tests, because other wise every `render-sync` would generate a different test files.

Reply:
https://github.com/openshift/cluster-node-tuning-operator/pull/594 issued a PR to fix that